vhdl_lang/src/analysis/names.rs

```diff
@@ -1845,7 +1845,10 @@ impl Declaration {
             Declaration::SubprogramBody(_) => "subprogram body",
             Declaration::Use(_) => "use",
             Declaration::Package(_) => "package instantiation",
+            Declaration::PackageDeclaration(_) => "package",
+            Declaration::PackageBody(_) => "package body",
             Declaration::Configuration(_) => "configuration",
+            Declaration::Disconnection(_) => "disconnection",
```

Comment:
I think calling this 'disconnection' as opposed to 'disconnect' could cause confusion. The rest of the names follow the names also given in the LRM, I think we should therefore call this 'disconnect' as well.

Reply:
The keyword is "disconnect" but LRM 7.4 is called "Disconnection Specification", similar to e.g. "Configuration Specification". Anyway I don't care which tbh.

Reply:
Ah, I didn't realize this. In this case, I think we should stick to the LRM like you proposed.